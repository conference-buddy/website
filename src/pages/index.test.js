import React from "react";
import IndexPage from "./index";
import { render, cleanup } from "@testing-library/react";

describe("<IndexPage />", () => {
  afterEach(cleanup);

  it("shows all elements", () => {
    const component = render(<IndexPage />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
