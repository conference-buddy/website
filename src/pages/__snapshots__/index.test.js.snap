// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexPage /> shows all elements 1`] = `
<DocumentFragment>
  <header>
    <nav
      class="MainNav px-sm-4 py-2 px-2 fixed-top d-flex justify-content-between MainNav-bg-out"
    >
      <div
        class="d-none d-sm-block d-flex align-items-center"
      >
        <a
          aria-current="page"
          class=""
          href="/"
        />
      </div>
    </nav>
  </header>
  <main>
    <div>
      <div
        class="IntroSection IntroSection-fullscreen-bg bg-clip-ellipse"
      >
        <div
          class="container d-flex align-items-center p-content-section "
        >
          <div
            class="text-center text-light"
          >
            <h1
              class="IntroSection-logo-font"
            >
              <strong>
                Conference Buddy
              </strong>
            </h1>
            <div
              class="lead py-2 px-3"
            >
              <p>
                Conference Buddy is for everyone who wants to attend a tech conference but is 
                <strong>
                  afraid to go alone
                </strong>
                . It's for everyone at tech events who is 
                <strong>
                  not comfortable being on their own
                </strong>
                 all the time. It’s for 
                <strong>
                  all you
                </strong>
                 first-timers, Introverts, socially awkwards, timid and shy people out there who are just like me!
              </p>
            </div>
          </div>
        </div>
        <div
          class="IntroSection-mouse-scroll-wrapper"
        >
          <div
            class="MouseScrollIcon d-none d-sm-block"
          >
            <a
              href="onPageLinkId-section-about"
            >
              <span
                class="MouseScrollIcon-mouse"
              >
                <span
                  class="MouseScrollIcon-wheel"
                />
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        class="container-fluid position-relative bg-bordered-wave"
      >
        <div
          class="p-content-section"
          id="onPageLinkId-section-about"
        >
          <div
            class="container"
          >
            <div
              class="d-none d-sm-block"
            >
              <h2
                class="display-4"
              >
                What is 
                <strong>
                  Conference Buddy
                </strong>
                ?
              </h2>
            </div>
            <div
              class="d-block d-sm-none"
            >
              <h2>
                What is Conference Buddy?
              </h2>
            </div>
            <div
              class="row pt-3"
            >
              <div
                class="col-md-6"
              >
                <h3>
                  What does it do?
                </h3>
                <p>
                  Conference Buddy provides a platform to find 
                  <strong>
                    companions for tech conferences
                  </strong>
                  . You can look for one or more person with whom you can attend a conference 
                  <strong>
                    together
                  </strong>
                  . Support each other, find a familiar face between all these strangers. You will feel more at ease and comfortable just knowing you’re not alone.
                </p>
                <p>
                  At the same time, you have the opportunity to 
                  <strong>
                    help others
                  </strong>
                   to feel better! Let’s make tech events more approachable - one Buddy at a time.
                </p>
              </div>
              <div
                class="col-md-6"
              >
                <h3>
                  Who can use it?
                </h3>
                <p>
                  That's pretty simple: 
                  <strong>
                    Everyone
                  </strong>
                   who feels like a bit of support at tech events would help them feel more at ease. At the same time it's for all you great folks out there who want's to be that kind of support!
                </p>
                <p>
                  It doesn't matter if it's your first time or if you're an international speaker: You feel nervous or anxious about being around a lot of (stranger) people and interacting with them? Conference Buddy wants to help you 
                  <strong>
                    feel more comfortable
                  </strong>
                  !
                </p>
                <p />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="container-fluid position-relative bg-bordered-wave"
      >
        <div
          class="p-content-section"
          id="onPageLinkId-section-about"
        >
          <div
            class="container"
          >
            <div
              class="d-none d-sm-block text-center"
            >
              <h2
                class="display-4"
              >
                Conference Buddy 2.0.
              </h2>
            </div>
            <div
              class="d-block d-sm-none text-center "
            >
              <h2>
                Conference Buddy 2.0.
              </h2>
            </div>
            <div
              class="pb-5 row justify-content-center"
            >
              <div
                class="col col-lg-8"
              >
                <h3
                  class="text-center"
                >
                  ✨ Webapp in progress ✨
                </h3>
                <p>
                  I'm actively working a release of a first MVP for a Conference Buddy web app! You can follow along the progress in the 
                  <a
                    href="https://github.com/conference-buddy/conference-buddy-web-app#readme"
                  >
                    github
                  </a>
                  !
                </p>
                <p>
                  If you want to be informed about the most important milestones - alpha release and, of course, the real release - you can sign up to my mailing list.
                </p>
              </div>
            </div>
            <div
              class="row col justify-content-center"
            >
              <form
                action="https://buttondown.email/api/emails/embed-subscribe/conferencebuddy"
                method="post"
                name="email-subscribe"
                target="popupwindow"
              >
                <label
                  class="sr-only"
                  for="bd-email"
                >
                  Enter your email
                </label>
                <div
                  class="input-group"
                >
                  <input
                    class="form-control"
                    id="bd-email"
                    name="email"
                    placeholder="Enter your email"
                    type="email"
                  />
                  <div
                    class="input-group-append"
                  >
                    <input
                      class="btn btn-outline-secondary"
                      type="submit"
                      value="Subscribe"
                    />
                  </div>
                </div>
                <p
                  class="text-center mt-2"
                >
                  <a
                    href="https://buttondown.email"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Powered by Buttondown.
                  </a>
                </p>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
  <footer>
    <div
      class="bg-light py-2 small text-center shadow-lg"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M383.9 308.3l23.9-62.6c4-10.5-3.7-21.7-15-21.7h-58.5c11-18.9 17.8-40.6 17.8-64v-.3c39.2-7.8 64-19.1 64-31.7 0-13.3-27.3-25.1-70.1-33-9.2-32.8-27-65.8-40.6-82.8-9.5-11.9-25.9-15.6-39.5-8.8l-27.6 13.8c-9 4.5-19.6 4.5-28.6 0L182.1 3.4c-13.6-6.8-30-3.1-39.5 8.8-13.5 17-31.4 50-40.6 82.8-42.7 7.9-70 19.7-70 33 0 12.6 24.8 23.9 64 31.7v.3c0 23.4 6.8 45.1 17.8 64H56.3c-11.5 0-19.2 11.7-14.7 22.3l25.8 60.2C27.3 329.8 0 372.7 0 422.4v44.8C0 491.9 20.1 512 44.8 512h358.4c24.7 0 44.8-20.1 44.8-44.8v-44.8c0-48.4-25.8-90.4-64.1-114.1zM176 480l-41.6-192 49.6 32 24 40-32 120zm96 0l-32-120 24-40 49.6-32L272 480zm41.7-298.5c-3.9 11.9-7 24.6-16.5 33.4-10.1 9.3-48 22.4-64-25-2.8-8.4-15.4-8.4-18.3 0-17 50.2-56 32.4-64 25-9.5-8.8-12.7-21.5-16.5-33.4-.8-2.5-6.3-5.7-6.3-5.8v-10.8c28.3 3.6 61 5.8 96 5.8s67.7-2.1 96-5.8v10.8c-.1.1-5.6 3.2-6.4 5.8z"
        />
      </svg>
       
      <a
        class="TextLink TextLink-primary "
        href="https://www.iubenda.com/privacy-policy/44138766"
      >
        Privacy Policy
      </a>
    </div>
    <div
      class="container-fluid small bg-gradient text-light text-center"
    >
      Made with 
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M462.3 62.6C407.5 15.9 326 24.3 275.7 76.2L256 96.5l-19.7-20.3C186.1 24.3 104.5 15.9 49.7 62.6c-62.8 53.6-66.1 149.8-9.9 207.9l193.5 199.8c12.5 12.9 32.8 12.9 45.3 0l193.5-199.8c56.3-58.1 53-154.3-9.8-207.9z"
        />
      </svg>
       lots of 
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 640 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M192 384h192c53 0 96-43 96-96h32c70.6 0 128-57.4 128-128S582.6 32 512 32H120c-13.3 0-24 10.7-24 24v232c0 53 43 96 96 96zM512 96c35.3 0 64 28.7 64 64s-28.7 64-64 64h-32V96h32zm47.7 384H48.3c-47.6 0-61-64-36-64h583.3c25 0 11.8 64-35.9 64z"
        />
      </svg>
       and an awesome 
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 576 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M528 448H48c-26.51 0-48-21.49-48-48V112c0-26.51 21.49-48 48-48h480c26.51 0 48 21.49 48 48v288c0 26.51-21.49 48-48 48zM128 180v-40c0-6.627-5.373-12-12-12H76c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm96 0v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm96 0v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm96 0v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm96 0v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm-336 96v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm96 0v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm96 0v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm96 0v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm-336 96v-40c0-6.627-5.373-12-12-12H76c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12zm288 0v-40c0-6.627-5.373-12-12-12H172c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h232c6.627 0 12-5.373 12-12zm96 0v-40c0-6.627-5.373-12-12-12h-40c-6.627 0-12 5.373-12 12v40c0 6.627 5.373 12 12 12h40c6.627 0 12-5.373 12-12z"
        />
      </svg>
    </div>
  </footer>
  <div
    style="position: fixed; bottom: 15px; right: 15px; cursor: pointer; transition-duration: 0.2s; transition-timing-function: linear; transition-delay: 0s; opacity: 0; visibility: hidden; transition-property: opacity, visibility;"
  >
    <svg
      class="text-dark"
      fill="currentColor"
      height="2em"
      stroke="currentColor"
      stroke-width="0"
      style="opacity: 0.5;"
      viewBox="0 0 448 512"
      width="2em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 432V80c0-26.51 21.49-48 48-48h352c26.51 0 48 21.49 48 48v352c0 26.51-21.49 48-48 48H48c-26.51 0-48-21.49-48-48zm355.515-140.485l-123.03-123.03c-4.686-4.686-12.284-4.686-16.971 0L92.485 291.515c-7.56 7.56-2.206 20.485 8.485 20.485h246.059c10.691 0 16.045-12.926 8.486-20.485z"
      />
    </svg>
  </div>
</DocumentFragment>
`;
